subprojects/common/junit-utils/src/main/java/ru/avito/util/RandomUtils.kt

```diff
@@ -5,11 +5,13 @@ package ru.avito.util
 import java.util.ArrayList
 import java.util.Date
 import java.util.Random
+import java.util.concurrent.ThreadLocalRandom
 
 // TODO: to consider using reflection for automation.
 // For example - https://github.com/mtedone/podam
 
-private val random = Random(System.currentTimeMillis())
+private val random: ThreadLocalRandom
```

Comment:
https://stackoverflow.com/questions/2546078/java-random-long-number-in-0-x-n-range